var/spack/repos/builtin/packages/amdscalapack/package.py

```diff
@@ -108,12 +128,9 @@ def cmake_args(self):
             c_flags.append("-Wno-incompatible-pointer-types")
             args.append(self.define("CMAKE_C_FLAGS", " ".join(c_flags)))
 
-        # link libflame library
-        args.extend(["-DLAPACK_LIBRARIES={0}".format(self.spec["lapack"].libs)])
-
         args.extend(
             [
-                "-DLAPACK_FOUND=true",
+                "-DUSE_OPTIMIZED_LAPACK_BLAS=true",
```

Comment:
What is this change about?

Reply:
Does this work with previous versions of `amdscalapack` or should we use a different option based on versions?